types/src/nmt.rs

```diff
@@ -91,6 +93,7 @@ pub type Proof = nmt_rs::simple_merkle::proof::Proof<NamespacedSha2Hasher>;
 ///  - secondary reserved namespaces - those use version `0xff` so they are always placed after
 ///    user-submitted data.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd, Hash)]
+#[cfg_attr(all(feature = "wasm-bindgen", target_arch = "wasm32"), wasm_bindgen)]
```

Comment:
I'm wondering how usage of namespace works in js, as we don't expose any methods iiuc

Reply:
good point, added getters for id and version as well as to_byte equivalent.

Reply:
so user wasn't able to create a namespace, added that too, together with blob creation